lib/v2/screens/explore/explore_screen.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:seeds/constants/app_colors.dart';
+import 'package:seeds/providers/notifiers/auth_notifier.dart';
+import 'package:seeds/providers/notifiers/members_notifier.dart';
+import 'package:seeds/design/app_theme.dart';
+import 'package:seeds/providers/notifiers/settings_notifier.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'package:seeds/v2/screens/explore/components/explore_info_card.dart';
+import 'package:seeds/v2/screens/explore/interactor/explore_bloc.dart';
+import 'package:seeds/v2/screens/explore/interactor/viewmodels/explore_events.dart';
+import 'package:seeds/v2/screens/explore/interactor/viewmodels/explore_state.dart';
+
+/// Explore SCREEN
+class ExploreScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
+      create: (context) => ExploreBloc()..add(LoadExplore(userName: SettingsNotifier.of(context).accountName)),
+      child: Scaffold(
+        backgroundColor: AppColors.primary,
+        body: Padding(
+          padding: const EdgeInsets.all(16.0),
+          child: BlocBuilder<ExploreBloc, ExploreState>(
+            builder: (context, ExploreState state) {
+              switch (state.pageState) {
+                case PageState.initial:
+                  return SizedBox.shrink();
+                case PageState.loading:
+                  return Container(child: Center(child: Text("Loading...", style: Theme.of(context).textTheme.headline3,)));
+                case PageState.failure:
```

Comment:
This is just for the moment, is it?  because erros willl be showed in a toast in the near future. I guess the page content on **failure state** will vary depending on the page.

Reply:
Yes, this is placeholder untill we define error and loading states